app/helpers/check_answers_helper.rb

```diff
@@ -28,6 +28,7 @@ def total_questions(subsection, case_log, form)
   def filter_conditional_questions(questions, case_log)
     applicable_questions = questions
 
+    # puts applicable_questions.count
```

Comment:
This looks like it was for debugging?

Reply:
Yep sorry got it

> Am I right in thinking this PR just bring in the about this log pages, it doesn't yet enable complex routing? (i.e. things like, if it's a relet 4-5 pages become unnecessary to show?)

It's to prove that the routing we have already is capable of any level of complexity